libs/langchain/langchain/tools/graphql/tool.py

```diff
@@ -21,21 +21,51 @@ class BaseGraphQLTool(BaseTool):
     If an error is returned with 'Bad request' in it, rewrite the query and try again.
     If an error is returned with 'Unauthorized' in it, do not try again, but tell the user to change their authentication.
 
-    Example Input: query {{ allUsers {{ id, name, email }} }}\
+    {example}
```

Comment:
nit: this description introduces leading whitespace

Reply:
Changed this around a bit in the followup changes.